src/lib.rs

```diff
@@ -331,12 +341,25 @@ impl Config {
         if !ndk {
             cxx_cfg.target(&target);
         }
+        let mut asm_cfg = cc::Build::new();
```

Comment:
Isn't this the same as `c_cfg` above? Should that be reused as the asm compiler?

Reply:
I'm under the impression that the asm compiler may or may not be the c compiler, depending on the system and the compiler used, although if you'd like me to combine them, I certainly can.

Reply:
C++ has a different configuration since it calls` cpp(true)`, but there's no equivalent for asm files so this'll just not work if the C compiler and assembler are different unfortunately